contributing/ci-omero.txt

```diff
@@ -67,46 +69,50 @@ OMERO jobs
 
 .. _deployment_servers:
 
-Deployment servers
-^^^^^^^^^^^^^^^^^^
+Deployment servers and web
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The table below lists all the hostnames, ports and URLs of the OMERO.web
 clients of the deployment jobs described above:
 
 .. list-table::
     :header-rows: 1
-    :widths: 10,20,20,10,40
+    :widths: 10,20,20,10,20,40
 
     -   * Series
-        * Deployment job
+        * Deployment job (server)
         * Hostname
         * Port
+        * Deployment job (web)
         * Webclient
 
     -   * DEV
         * :term:`OMERO-DEV-merge-deploy`
         * eel.openmicroscopy.org
         * 4064
-        * https://eel.openmicroscopy.org/merge
+        * :term:`WEB-DEV-merge-deploy`
+        * http://web-dev-merge.openmicroscopy.org
 
     -   * DEV
         * :term:`OMERO-DEV-latest-deploy`
         * eel.openmicroscopy.org
         * 14064
-        * https://eel.openmicroscopy.org/latest
+        * :term:`WEB-DEV-latest-deploy`
+        * http://web-dev-latest.openmicroscopy.org
 
     -   * DEV
         * :term:`OMERO-DEV-merge-integration`
         * eel.openmicroscopy.org
         * 24064
-        * https://eel.openmicroscopy.org/integration
+        * :term:`WEB-DEV-integration-deploy`
+        * http://web-dev-integration.openmicroscopy.org
 
     -   * Breaking
         * :term:`OMERO-DEV-breaking-deploy`
         * trout.openmicroscopy.org
         * 34064
-        * https://trout.openmicroscopy.org/breaking
-
+        * :term:`WEB-DEV-breaking-deploy`
+        * http://web-dev-breaking.openmicroscopy.org
```

Comment:
At http://web-dev-breaking.openmicroscopy.org/webclient/login/ the server list there does not include 'trout' as would be expected from all the other 3 logins above. Would be nice to see 'trout' in that list so we know where it's connecting to.